product-scenarios/4-single-sign-on/4.4-application-management/4.4.1-soap-apis/src/test/java/org/wso2/identity/scenarios/test/ApplicationManagementTestCase.java

```diff
@@ -79,7 +81,7 @@ public void testInit() throws Exception {
         applicationManagementServiceClient = new ApplicationManagementServiceClient(sessionCookie, backendServiceURL,
                 configContext);
         userMgtClient = new UserManagementClient(backendServiceURL, sessionCookie);
-        userMgtClient.addRole(IDP_ROLE_1, new String[0], new String[]{PERMISSION_ADMIN_LOGIN});
+//        userMgtClient.addRole("Internal/"+IDP_ROLE_1, new String[0], new String[]{PERMISSION_ADMIN_LOGIN});
```

Comment:
why commented here

Reply:
We are adding new role here. Re-using internal(admin role)

Reply:
if this commented lines are not used, let's remove them

Reply:
Fixed with https://github.com/wso2/product-is/pull/14317/commits/1b4ab737ecfa0d7092d2c697a13b8ad39d9ad538